config/jobs/kubernetes/sig-cloud-provider/gcp/gce-conformance.yaml

```diff
@@ -23,6 +23,7 @@ periodics:
       - --scenario=kubernetes_e2e
       - --
       - --extract=ci/latest-fast
+      - --extract-ci-bucket=k8s-release-dev
```

Comment:
@justaugustus do we always want to get version markers from `k8s-release-dev` now, or are some jobs (not necessarily owned by sig-release) still requiring usage of `kubenertes-release-dev`?

Reply:
@hasheddan -- explained in the updated description, but pasting to fire the email notification:

> As we continue to migrate release-blocking jobs to a dedicated K8s Infra
cluster, jobs that use the latest-fast marker need to extract builds
from gs://k8s-release-dev, which is the K8s Infra equivalent of
gs://kubernetes-release-dev.
>
> A new flag (--extract-ci-bucket=k8s-release-dev) was added to support
this transitional use case, so we employ it here.

Reply:
@justaugustus gotcha 👍 was just wondering if we were ready to modify the extract logic to use this by default

Reply:
@hasheddan -- Not yet. The primary (non-fast) build jobs still need to be migrated. That's being tracked in https://github.com/kubernetes/test-infra/issues/19483.